app/agents_v2/primary_agent/reasoning/reasoning_engine.py

```
@@ -16,6 +16,7 @@
 from langchain_core.messages import BaseMessage, HumanMessage, SystemMessage
 from opentelemetry import trace
 
+from app.core.settings import settings
```

Ruff (0.12.2)

`app.core.settings.settings` imported but unused

Remove unused import: `app.core.settings.settings`

(F401)